modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/AccountResourceDTOConverter.java

```diff
@@ -273,14 +274,15 @@ private AccountContactInformation _toAccountContactInformation(
 			AccountEntry accountEntry, DTOConverterContext dtoConverterContext)
 		throws Exception {
 
-		if (!_accountEntryModelResourcePermission.contains(
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-10855") ||
```

Comment:
This feature flag doesn't exist anymore, we removed it so this reuse of it is not possible. Which is the error thrown with this code? We need to solve it compared the use of the old FF